pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>io.vlingo</groupId>
   <artifactId>vlingo-actors</artifactId>
-  <version>0.8.3</version>
+  <version>0.9.0</version>
```

Comment:
My only comment is that we have a release contract that increments the version number across the whole platform to keep all artifacts at the same level. Did you have a specific intention in using `0.9.0`?

Reply:
I will change the version to 0.8.4.